src/mono/mono/culture/empty.txt

```diff
@@ -0,0 +1 @@
+This directory intentionally empty in the dotnet/runtime repo
```

Comment:
this should be replaced with a stub Makefile like we do for btls/dis: https://github.com/dotnet/runtime/blob/master/src/mono/mono/dis/Makefile

you'll need to force add since Makefile is in the .gitignore.